models/user.js

```diff
@@ -0,0 +1,52 @@
+module.exports = function(sequelize, DataTypes) {
+  var User = sequelize.define("User", {
+    fullname: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        len: [1, 100]
+      }
+    },
+    age: {
+      type: DataTypes.TINYINT,
+      isInt: true,
+    },
+    gender: {
+      type: DataTypes.STRING,
+      validate: {
+        len: [1, 25]
+      }
+    },
+    googleID: {
+      type: DataTypes.STRING,
+      required: true,
+      validate: {
+        len: [1, 100]
+      }
+    },
+    emailAddress: {
+        type: DataTypes.STRING,
+        validate: {
+            isEmail: true,
+            len: [1, 255]
+        }
+    },
+    profilePhoto: {
+        type: DataTypes.TEXT,
+        // isUrl: true,
+    },
+    created_at: {
```

Comment:
I think we can delete this "created_at" column, since Sequelize will handle that for us